### ISSUE TITLE
Bug in interpData

### ISSUE THREAD

**Issue by matteodefelice** [SantanderMetGroup/downscaleR#15]
I'm using the last devel version from github.
```
> data(iberia_ncep_ta850)
> par(mfrow = c(2,1))
> plotMeanGrid(iberia_ncep_ta850)
> # Bilinear interpolation to domain centered in Spain using a 0.5 degree resolution
> # in both X and Y axes
> t <- interpData(iberia_ncep_ta850, new.coordinates = list(x = c(-10,5,.5),
+                                                           y = c(36,44,.5)),
+                 method = "bilinear")
[2016-05-06 11:45:23] Performing multi-member bilinear interpolation... may take a while
[2016-05-06 11:45:23] Interpolating member 1 out of 1
Error in z[cbind(lx1 + 1, ly1)] : subscript out of bounds
```

**Comment by jbedia** [comment]
Hi Matteo,
I am right now doing changes. The branch is not stable at this point. I 
will let you know as soon as interpData is ready.
Thanks!

**Comment by jbedia** [comment]
Hi Matteo. I've just pushed a new version of interpData. Still needs a lot of polishing, and it will probably crash for station interpolation (need some more time to update), but it works for grids as far as I have tested. The most remarkable change from the user point of view is the dramatical speed-up of nearest-neighbour option, that was previously almost unusable for large multimember grids. I look forward to hearing your feedback. Best regards
